src/lambda_function/raw/template.yaml

```diff
@@ -52,7 +52,7 @@ Resources:
       Handler: app.lambda_handler
       Runtime: !Sub "python${LambdaPythonVersion}"
       Role: !Ref RoleArn
-      MemorySize: 1024
+      MemorySize: 1769
```

Comment:
Nit: Something i was thinking of but do we want to make this a variable instead? So the develop version of the lambda can declare a different memory size than the prod version? Or were you running into this for both?

Reply:
I don't think so. This is not an expensive function to run, so no harm in keeping the configurations the same.